pip_deps/jetson_requirements.txt

```diff
@@ -1,3 +0,0 @@
-numpy==1.16.2

```

Comment:
not sure how this happened, a pip_deps/onboard_requirements file exists, plz get rid of it